lib/app.js

```diff
@@ -0,0 +1,39 @@
+"use strict";
```

Comment:
Agreed. I typically just call the main entry point for Node projects "app.js", so I did it that way. I'll change it to "index.js".

I'll also look into using Dizzy.